texts/common/dummy.ts

```diff
@@ -51,3 +51,5 @@ export const animalTherapy: Event = {
   tags: 'general',
   cca: 'Raffles Volunteer Corps',
 }
+
+export const dummyData:Event[] = [valentinesFundraiser, animalTherapy];
```

Comment:
```suggestion
export const dummyData:Event[] = [valentinesFundraiser, animalTherapy];

```
newline at end of file... ensure you have Prettier configured to save on format on your IDE ya